includes/admin/class-sensei-status.php

```diff
@@ -0,0 +1,276 @@
+ * @since 3.7.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Sensei_Status class.
+ *
+ * @since 3.7.0
+ */
+class Sensei_Status {
+	/**
+	 * Instance of class.
+	 *
+	 * @var self
+	 */
+	private static $instance;
+
+	/**
+	 * Sensei_Status constructor. Prevents other instances from being created outside of `self::instance()`.
+	 */
+	private function __construct() {}
+
+	/**
+	 * Fetches an instance of the class.
+	 *
+	 * @return self
+	 */
+	public static function instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Adds all filters and actions.
+	 *
+	 * @since 3.7.0
+	 */
+	public function init() {
+		add_filter( 'debug_information', [ $this, 'add_sensei_debug_info' ] );
+		add_filter( 'site_status_tests', [ $this, 'add_sensei_tests' ] );
+	}
+
+	/**
+	 * Adds Sensei Debug information to the Site Health > Info screen.
+	 *
+	 * @param array $info Information to show.
+	 *
+	 * @return array
+	 */
+	public function add_sensei_debug_info( $info ) {
+		$section = [
+			'label'  => __( 'Sensei LMS', 'sensei-lms' ),
+			'fields' => [],
+		];
+
+		$section['fields']['template_overrides']     = $this->get_template_overrides_info();
+		$section['fields']['is_calculation_pending'] = $this->get_is_calculation_pending_info();
+		$section['fields']['legacy_enrolment']       = $this->get_legacy_enrolment_info();
+
+		$info['sensei-lms'] = $section;
+
+		return $info;
+	}
+
+	/**
+	 * Get legacy enrolment info field.
+	 *
+	 * @return array
+	 */
+	private function get_legacy_enrolment_info() {
+		$legacy_enrolment_timestamp    = get_option( 'sensei_enrolment_legacy' );
+		$value_is_legacy_enrolment_set = __( 'Not applicable', 'sensei-lms' );
+
+		if ( $legacy_enrolment_timestamp ) {
+			$installed_time = wp_date( get_option( 'date_format' ) . ' ' . get_option( 'time_format' ), $legacy_enrolment_timestamp );
+			// translators: Placeholder is datetime for when the instance was upgraded.
+			$value_is_legacy_enrolment_set = sprintf( __( 'Yes, updated Sensei LMS from a pre-3.0 version at %s', 'sensei-lms' ), $installed_time );
+		}
+
+		return [
+			'label' => __( 'Legacy enrollment migration', 'sensei-lms' ),
+			'value' => $value_is_legacy_enrolment_set,
+			'debug' => $legacy_enrolment_timestamp,
+		];
+	}
+
+	/**
+	 * Get template overrides info field.
+	 *
+	 * @return array
+	 */
+	private function get_template_overrides_info() {
+
+		$template_overrides_value = __( 'No template overrides', 'sensei-lms' );
+		$template_overrides       = $this->get_template_override_status();
+
+		if ( ! empty( $template_overrides ) ) {
+			$template_overrides_value = [];
+			foreach ( $template_overrides as $template => $versions ) {
+				if ( $versions['sensei_version'] !== $versions['theme_version'] ) {
+					// translators:: First placeholder is Sensei LMS' template version and the second placeholder is the theme's version.
```

Comment:
There's an extra `:` here

Reply:
Removed it in 4f9038370d4cee9fa86dfa590c94eecfd83deb49